pkg/handler/collector/deps_dev/deps_dev.go

```diff
@@ -302,11 +308,20 @@ func (d *depsCollector) collectAdditionalMetadata(ctx context.Context, pkgType s
 
 func getVersionKey(pkgType string, namespace *string, name string, version *string) (*pb.VersionKey, error) {
 	queryName := ""
-	if namespace != nil && *namespace != "" {
-		queryName = strings.TrimSuffix(*namespace, "/") + "/" + name
+	if pkgType != "maven" {
```

Comment:
is it worth creating an issue with label "data-quality" to talk about this? or is this specific to deps.dev?

Reply:
This is specific to deps.dev